src/Controls/tests/DeviceTests/Elements/CollectionView/CollectionViewTests.cs

```diff
@@ -45,8 +45,15 @@ public async Task ItemsSourceDoesNotLeak()
 
 			IList logicalChildren = null;
 			WeakReference weakReference = null;
+
+			// Recently discovered that Android CollectionView is leaking when given a header or footer
+			// See: 
```

Comment:
Maybe forgot link? Otherwise, looks good for now. 👍

Reply:
Whoops :P